File: Benchmarking/src/gen_103_9.cpp
  #include <vector>
  #include <iostream>
  #include <chrono>
  double f39(const double &v0) {
    double v1 = v0;
    double v6 = v1;
    const std::vector<double> v3 { v6 };
    const double v5 = v3[0];
    std::vector<double> v4 = v3;
    double v8 = v0;
    const double v10 = v4[0];
    std::vector<double> v2 = v4;
    const double v12 = v4[0];
    std::vector<double> v14 = v2;
    std::vector<double> v26 = v4;
    std::vector<double> v20 = v2;
    const double v13 = v14[0];
    v2[0] = v13;
    v2[0] = v10;
    const double v16 = v2[0];
    v26[0] = v16;
    const std::vector<std::vector<double>> v40 { v26, v20, v3 };
    v20[0] = v12;
    v14[0] = v5;
    const std::vector<double> v41 = v40[1];
    const double v49 = v41[0];
    v2[0] = v13;
    v14[0] = v8;
    return v49;
  }
  __attribute__((noinline))
  double f0(const std::vector<std::vector<double>> &v0, const double &v1) {
    const std::vector<double> v7 = v0[0];
    double v2 = v1;
    std::vector<double> v10 = v7;
    v10[0] = v1;
    v10[0] = v1;
    v10[0] = v2;
    const double v23 = v10[0];
    v10[0] = v23;
    const std::vector<double> v17 = v0[0];
    const double v36 = v10[0];
    const double v33 = v17[0];
    const double v40 = v33 / v36;
    const double v50 = f39(v40);
    return v50;
  }
  int main() {
    std::vector<std::vector<double>> v0({ { 0.0 }, { 1.0 }, { 2.0 } });
    double v1(3.0);
    auto start = std::chrono::high_resolution_clock::now();
    double result;
    for (int i = 0; i < 1000; i ++) {
      result = f0(v0, v1);
    }
    auto end = std::chrono::high_resolution_clock::now();
    double value = *((double*) &result);
    std::cout << value << "\n";
    std::cout << std::chrono::duration_cast<std::chrono::nanoseconds>(end-start).count();
    std::cout << "\n";
    return 0;
  }

File: Benchmarking/src/gen_103_9.swift
  import Dispatch
  func f39(_ v0: Double) -> Double {
    var v1: Double = v0
    var v6: Double = v1
    let v3: [Double] = [v6]
    let v5: Double = v3[0]
    var v4: [Double] = v3
    var v8: Double = v0
    let v10: Double = v4[0]
    var v2: [Double] = v4
    let v12: Double = v4[0]
    var v14: [Double] = v2
    var v26: [Double] = v4
    var v20: [Double] = v2
    let v13: Double = v14[0]
    v2[0] = v13
    v2[0] = v10
    let v16: Double = v2[0]
    v26[0] = v16
    let v40: [[Double]] = [v26, v20, v3]
    v20[0] = v12
    v14[0] = v5
    let v41: [Double] = v40[1]
    let v49: Double = v41[0]
    v2[0] = v13
    v14[0] = v8
    return v49
  }
  @inline(never)
  func f0(_ v0: [[Double]], _ v1: Double) -> Double {
    let v7: [Double] = v0[0]
    var v2: Double = v1
    var v10: [Double] = v7
    v10[0] = v1
    v10[0] = v1
    v10[0] = v2
    let v23: Double = v10[0]
    v10[0] = v23
    let v17: [Double] = v0[0]
    let v36: Double = v10[0]
    let v33: Double = v17[0]
    let v40: Double = v33 / v36
    let v50: Double = f39(v40)
    return v50
  }
  func benchmark() {
    let v0: [[Double]] = [[0.0], [1.0], [2.0]]
    let v1: Double = 3.0
    let start = DispatchTime.now().uptimeNanoseconds
    var result: Double = 3.0
    for _ in 1...1000 {
      result = f0(v0, v1)
    }
    let end = DispatchTime.now().uptimeNanoseconds
    print(result)
    print(end - start)
  }
  benchmark()
